packages/safe-tools-client/app/helpers/payment-error-message.ts

```diff
@@ -0,0 +1,28 @@
+import { helper } from '@ember/component/helper';
+
+type PositionalArgs = [string];
+
+interface Signature {
+  Args: {
+    Positional: PositionalArgs;
+  };
+  Return: string;
+}
+
+export function paymentErrorMessage([failureReason]: PositionalArgs) {
+  // Try to match the error message to a known error explained in a way that makes sense to the user
+  if (failureReason) {
+    if (failureReason.includes('InvalidPeriod'))
+      return 'payment attempted at the wrong time'; // Could be the crank's fault. Not much the user can do about it except report it to support
```

Comment:
Not sure what to tell the user here, looking for ideas. It's actually quite awkward to say that the payment was attempted at the wrong time since this is the core feature of this tool. Maybe this should go into the "unknown error" category instead and we urge the user to contact support, especially since we don't expect this error to happen often, only if there is a scheduling misconfiguration at our side.